ESMF/GOCART2G_GridComp/SU2G_GridComp/SU2G_GridCompMod.F90

```diff
@@ -1120,10 +1122,10 @@ subroutine Run2 (GC, import, export, clock, RC)
 
 !if(mapl_am_i_root()) print*,'SU2G Run2 WetRemoval sum(xh2o2) = ',sum(xh2o2)
 !if(mapl_am_i_root()) print*,'SU2G Run2 WetRemoval sum(self%h2o2_init) = ',sum(self%h2o2_init)
-
+!   Certain variables are multiplied by 1.0e-9 to convert from nanometers to meters
     call SU_Compute_Diags ( self%km, self%klid, self%radius(nSO4), self%sigma(nSO4), self%rhop(nSO4), &
                             MAPL_GRAV, MAPL_PI, nSO4, self%diag_MieTable(self%instance), &
-                            self%diag_MieTable(self%instance)%channels, &
+                            self%diag_MieTable(self%instance)%channels*1.0e-9, &
```

Comment:
I would not bother in this PR, but probably in the next changes, a named parameter like   NANOMETERS_PER_METER would be useful to make these factors of a billion more obvious to pure sw developers.

Reply:
This input is used in the call to Chem_MieQuery within SU_Compute_Diags. During today's meeting Arlindo ask that we change Chem_MieQuery to work in units of nanometers instead of meters. This would allow us to remove these types of unit conversions in the call.

Reply:
We should stick to SI units in the code. Conversions like these are fine because they make the resource files more user friendly. What Tom suggested is to replace the hardcoded numerical value with a parameter/const variable to make the code more readable.

Reply:
I do like the idea of sticking with SI units in the code. I thought Arlindo asked me to change Chem_MieQuery to work in nanometers in our meeting today? I must have misunderstood him.

Reply:
My impression was that he requested a change of the 'monochromatic wavelengths' label, i.e. add 'nm' at the end of it, to make it similar to the other two 'wavelength' specific labels.